### ISSUE TITLE
allow to drop .json.gz files

### ISSUE THREAD

**Issue by mariemoinet** [nextstrain/auspice.us#112]
Some tools like UCSC UShER provide .json.gz files "ready" to be uploaded in Auspice... but .gz files don't seem to be accepted. A feature to accept .json.gz and unzip them directly in Auspice would be very handy.

**Comment by jameshadfield** [comment]
Yeah, this seems like a sensible addition for us to make. AFAIK we can't leverage the browser's decompression to do this and will have to use something like `JSZip`, but I don't have a problem with that for `auspice.us`.
